src/main/java/org/openstreetmap/atlas/geography/atlas/bloated/BloatedArea.java

```diff
@@ -137,6 +137,11 @@ public String toString()
                 + ", tags=" + this.tags + ", relationIdentifiers=" + this.relationIdentifiers + "]";
     }
 
+    public BloatedArea withAddedTag(final String key, final String value)
+    {
+        return withTags(BloatedEntity.addNewTag(getTags(), key, value));
```

Comment:
In `BloatedEntity.addNewTag(..)` - do we care about overriding an existing tag? The naming implies adding a new one, but we don't differentiate between an addition or replacement. Should we log something if we override?

Reply:
This is a good point. We also have a `withReplacedTag` method, so I can see how this might be confusing. I think we could either:

1) Remove `withReplacedTag` all together, and just use `withAddedTag` in all cases (having it log a warning if it replaces a tag).
2) Change `withAddedTag` to do nothing if the tag already exists (and log an error/warn)

Curious what you think of `1` or `2`.

Reply:
I'm leaning towards 1. - using `withAddedTag` and `removeTag`, with each method logging a warning if there is a replacement or if there tag being removed doesn't exist. This is also consistent with the `ChangeType` model of representing modify operations as adds.

Reply:
Sounds good to me!

Reply:
Ok, will do that in a future PR.